.github/workflows/task-get-linux-configurations.yml

```diff
@@ -9,11 +9,13 @@ env:
                     'rockylinux:8',
                     'rockylinux:9',
                     'debian:bullseye',
+                    'debian:bookworm',
                     'amazonlinux:2',
                     'mcr.microsoft.com/cbl-mariner/base/core:2.0']"
   ALL_ARM_IMAGES: "['ubuntu:jammy',
                     'ubuntu:focal',
-                    'ubuntu:bionic']"
+                    'ubuntu:bionic',
+                    'debian:bookworm']"
```

Comment:
Do we need arm for `bookworm`?
Sure we don't need `bullseye` here as well?

Reply:
`bullseye` changes are not listed in the epic MOD-7304, but it is part of the PRD.
Additionally, we don't have json package for bullseye on arm64, then maybe we should create a new PR to support that OS/architecture.